app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -965,6 +976,18 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
         (activity as ChatRoomActivity).showToolbarTitle(toolbarTitle)
     }
 
+    override fun unscheduleDrawable(who: Drawable?, what: Runnable?) {
+        text_message?.removeCallbacks(what)
+    }
+
+    override fun invalidateDrawable(p0: Drawable?) {
+        text_message?.invalidate()
+    }
+
+    override fun scheduleDrawable(who: Drawable?, what: Runnable?, w: Long) {
```

Comment:
What `w` (as a Long) stands for?

Reply:
w stands for when, which is a reserved word on kotlin

Reply:
Go with `when_` or `_when`. It's more meaningful than declaring only `w` IMO